Tests.Unit.Core.ApplicationServices/ApplicationServices/Organizations/OrganizationUnitServiceTest.cs

```diff
@@ -271,6 +271,22 @@ public void TransferSelectedOrganizationRegistrations_Returns_Forbidden(bool isU
             Assert.Equal(OperationFailure.Forbidden, result.Value.FailureType);
         }
 
+        [Fact]
+        public void GetAccessRightsByOrganization_Returns_NotFound_When_Organization_Was_NotFound()
+        {
+            //Arrange
+            var orgUuid = A<Guid>();
```

Comment:
what about the success case? I would assume you have already tested them using the GetAccess method tests, but we need at least one proof of it in this context as well. just testing "not found" is the poor mans choice